packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,212 @@
+| autoFocus                                                                                          | `boolean`                         | No                                                                                  | `false`                                                   | The autofocus global attribute is a Boolean attribute indicating that an element should be focused on page load. [Web Reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus), [Native Reference](https://reactnative.dev/docs/textinput#autofocus) |
+| fullWidth                                                                                          | `boolean`                         | No                                                                                  | `false`                                                   | **Discuss if we want to add support for full-width input fields.**                                                                                                                                                                                                                    |
+| autofillSuggestionType                                                                             | `password`, `passwordNew`, `none` | No                                                                                  |                                                           | Specifies autocomplete hints for the system, so it can provide autofill                                                                                                                                                                                                               |
+| keyboardReturnKeyType. [Check this table for full reference](#return-key-types-for-web-and-native) | `go`, `done`, `next`, `send`      | No                                                                                  | Closest based on the `type` attribute                     | Determines how the return key should look on the keyboard on mobile devices or virtual keyboard                                                                                                                                                                                       |
+
+## Notes
+
+Reusable components candidate
+
+- label
+- helpText/errorText/successText
+- character counter(only input field)
+- clear icon button
+- prefix/suffix
+
+## Open questions
+
+- [ ] Do we have a usecase of using textfield without label or form level things?
+- [ ] Do we need to expose `autoCapitalize` for native?
+- [ ] Do we need to expose `onSubmitEditing` for native?
+- [ ] Do we need to fix the height for reserving space for help/error/success text?
```

Comment:
I think we should not fix the height and let this wrap, there could be sensitive/important information as part of an input field validation that would need to be displayed fully.

Reply:
pending on design, saurav will test this visually and conclude